<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../images/393883823_3566576113658158_9212498078691773248_n.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="afficher">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>AFFICHER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>AJOUTER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
padding: 5px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
   
    border-radius: 5px;
  
font: 700 11pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>DATE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quant">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 15px;
    
	font: 700 9pt &quot;Segoe UI&quot;;
    
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/426963771_940106313792500_9064911174535547542_n.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="idf">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 15px;
    
	font: 700 9pt &quot;Segoe UI&quot;;
    
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="recherche">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  background-color: rgba(255, 255, 255, 0.5); 
padding: 5px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color: #fff;
    border: 2px solid #43aff7;
    border-radius: 15px;
    font-size: 14px;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>        recherche :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prix">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>340</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 15px;
    
	font: 700 9pt &quot;Segoe UI&quot;;
    
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: transparent; 
padding: 5px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
   
    border-radius: 5px;
   
font: 700 11pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>IDF:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>580</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>LOGOUT</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 700 11pt &quot;Segoe UI&quot;;
background-color: transparent;
padding: 5px;
    
    border-radius: 5px;
</string>
    </property>
    <property name="text">
     <string>QUANTITE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>56</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/423903620_1378110712823805_3453213355354231890_n.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
padding: 5px;
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
    border-radius: 5px;
 
font: 700 11pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>PRIX_PR:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>100</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/pk.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="filtrer">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>TRIE</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="musique">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>580</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0.5); 

padding: 5px;

    border: 2px solid #43aff7;
    border-radius: 5px;
    font-size: 14px;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/360_F_301439209_vpF837oCGM1lp0cnC7stzCBn3th0dQ6O.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>100</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/pdf.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_9">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 20pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);
padding: 5px;
   
    border-radius: 5px;
   
font: 700 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>IDC:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>331</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="statButton">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>520</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>STAT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pdf">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>90</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>       PDF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idc">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 15px;
    
	font: 700 9pt &quot;Segoe UI&quot;;
    
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
font: 700 16pt &quot;Segoe UI&quot;;
    border: none;
</string>
    </property>
    <property name="text">
     <string>Gestions des Commandes :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/426721553_3308243649479444_9142994978690348616_n.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/426833070_362133560103314_8738250760355801066_n.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    padding: 5px;
    border: 2px solid #43aff7;
    border-radius: 15px;
    
	font: 700 9pt &quot;Segoe UI&quot;;
    
	color: rgb(255, 255, 255);
}

QLineEdit:focus {
    border-color: #2ecc71; /* Couleur de la bordure lorsqu'il est activé */
}
</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>521</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="update">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>520</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}
</string>
    </property>
    <property name="text">
     <string>MODIFIER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="supprimer">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>520</y>
      <width>158</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0.3);
    border: 0px solid rgba(255, 255, 255, 0.3);
  border-radius: 20px;
  padding: 8px 16px;
  color: #fff;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton:hover {

background-color: rgba(255, 255, 255, 0.5);
}

</string>
    </property>
    <property name="text">
     <string>SUPPRIMER</string>
    </property>
   </widget>
   <widget class="QLabel" name="qrCodeLabel">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(148, 148, 148);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>afficher</zorder>
   <zorder>ajouter</zorder>
   <zorder>recherche</zorder>
   <zorder>logout</zorder>
   <zorder>label_5</zorder>
   <zorder>label_10</zorder>
   <zorder>musique</zorder>
   <zorder>statButton</zorder>
   <zorder>pdf</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>textBrowser</zorder>
   <zorder>lineEdit_14</zorder>
   <zorder>lineEdit_11</zorder>
   <zorder>lineEdit_7</zorder>
   <zorder>lineEdit_9</zorder>
   <zorder>lineEdit_13</zorder>
   <zorder>idc</zorder>
   <zorder>quant</zorder>
   <zorder>prix</zorder>
   <zorder>idf</zorder>
   <zorder>label_7</zorder>
   <zorder>label_12</zorder>
   <zorder>filtrer</zorder>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>date</zorder>
   <zorder>tableView</zorder>
   <zorder>update</zorder>
   <zorder>supprimer</zorder>
   <zorder>qrCodeLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
